torchtune/training/_grad_scaler.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Does this really need its own file?

Reply:
No

Reply:
Where do you wanna put it then? Otherwise I am gonna copy-paste this in every recipe which is worse imo